index.html

```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+        <!-- <section class="box header">
+        <div class="top_section">
+            <p class="time">9:41</p>
+            <ul>
+                <li><img src="Media/Cellular Connection.png" alt="Network Cell"></li>
+                <li><img src="Media/Wifi.png" alt="wifi"></li>
+                <li><img src="Media/Battery.png" alt="Battery"></li>
+            </ul> 
+        </div>
+        <nav class="header">
+            <a href="#" class="logo">
+                <h2>My Logo</h2>
+            </a>
+            <a href="#"><img src="Media/Union.svg" alt="Menu" class="menu_item2"></a> -->
+            <!-- <nav class="menu_icon">
+                <a href="#" class="Logo"><h2>My Logo</h2></a>
+                    <a href="#" class="menu_item2"><img src="Media/Union.svg" alt="Menu"></a>
+            </nav> -->
+            <header class="header">
+               <a href="#" class="logo">
+                <h2>My Logo</h2>
+               </a> 
+               <a href="#"><img src="Media/Union.svg" alt="Menu" class="menu_iteme2">></a>
```

Comment:
- [ ] As mentioned by the previous reviewer #1 :point_left: I think it would be better if this section has the `margins (top, bottom, left, right)` defined in the Figma design, currently, the `margins` of the `header` appear to be inconsistent with the Figma design. Please see the screenshot below :point_down:  for a comparison of your design and Figmad design.

| Your Design | Figma Design |
|--------|--------|
| ![your-design](https://user-images.githubusercontent.com/16392046/235266737-3e8dbe72-9711-4780-9705-1282a746acc3.png) | ![figma-design](https://user-images.githubusercontent.com/16392046/235266974-7be79375-cc4e-44db-80fe-f9273f4561fd.png) |